PrivacyPolicy.md

```diff
@@ -0,0 +1,50 @@
+# Information Collected And Transmitted By Git Extensions
+
+First, a reminder: Git Extensions is provided "as is", without warranty of any kind, express or implied, including but not limited to the warranties of merchantability, fitness for a particular purpose and noninfringement.
+
+With that out of the way, here's a breakdown of all the information we may collect via Application Insights.
+
+### Application-Level
+Includes:
+* Exception information
+  * Could, in rare cases, contain paths packages on your computer
+* Machine name
+* Host name
```

Comment:
I cannot see how names of machine and host could help tracking down anything (data minimization!).

Reply:
I've looked through the logs and I don't see them logged

Reply:
Good. Then the regarding three lines could be removed.

Reply:
You have already removed these two of the three lines but kept a copy of the original section "Application-Level".